yarn.lock

```diff
@@ -78,7 +78,7 @@
     semver "^5.4.1"
     source-map "^0.5.0"
 
-"@babel/core@^7.0.0", "@babel/core@^7.1.0", "@babel/core@^7.1.6", "@babel/core@^7.2.0", "@babel/core@^7.3.4":
+"@babel/core@^7.0.0", "@babel/core@^7.1.0", "@babel/core@^7.1.6", "@babel/core@^7.2.0":
```

Comment:
Thanks, looks good. The only thing is, that after running `yarn install` I'm getting 12 mostly Babel related differences in `yarn.lock`. Looks like the problem we faced before.

<img width="735" alt="Screenshot 2019-04-03 at 10 12 01" src="https://user-images.githubusercontent.com/2660330/55463810-15f3e080-55fa-11e9-94b0-6ba56c545381.png">

Reply:
Can you try again? Should be fixed now

Reply:
Yes, it's ok now. 🚀

Reply:
👍 I will merge it when CI finishes

Reply:
Definitely, thanks.